USAGE.md

```diff
@@ -496,3 +498,106 @@ Get-GitHubEvent -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Issue
 ```powershell
 Get-GitHubEvent -OwnerName Microsoft -RepositoryName PowerShellForGitHub -EventID 1
 ```
+
+----------
+
+### Advanced
+
+#### Migrating blog comments to github issues
+
+I used this module to migrate my blog comments from Disqus to Github Issues. I'm using Utterances App to display the Github Issues on each blog posts. [See this blog post for more details](https://lazywinadmin.com/2019/04/moving_blog_comments.html)
```

Comment:
I prefer to keep MarkDown files limited to ~80-100 characters per line, in order to reduce the need for horizontal scrolling when opened up in a simple text editor.  MarkDown renderers will skip forced linebreaks and still render the content with proper wrapping.